mne/time_frequency/tfr.py

```diff
@@ -1970,6 +2038,9 @@ def plot(
         want_shape[ch_axis] = len(idx_picks) if combine is None else 1
         want_shape[freq_axis] = len(freqs)  # in case there was fmin/fmax cropping
         want_shape[time_axis] = len(times)  # in case there was tmin/tmax cropping
+        want_shape = [
+            n for i, n in enumerate(want_shape) if self._dims[i] != "taper"
```

Comment:
Perhaps equivalent and cleaner?
```suggestion
            n for dim, n in zip(self._dims, want_shape) dim != "taper"
```